Error while generating required QSC certificates

Hi, I am getting the following error while generating all required QSC certificates

```
ubuntu@ubuntu:~/oqs-demos/nginx/fulltest$ python3 genconfig.py
. > /tmp/opt/openssl/apps/openssl req -x509 -new -newkey rsa:4096 -keyout root/CA.key -out root/CA.crt -nodes -subj /CN=oqstest_CA -days 500 -config /tmp/opt/openssl/apps/openssl.cnf
New root cert residing in root/CA.crt.
. > /tmp/opt/openssl/apps/openssl ecparam -name prime256v1 -out pki/prime256v1.pem
Can't open pki/prime256v1.pem for writing, No such file or directory
0:error:02001002:system library:fopen:No such file or directory:crypto/bio/bss_file.c:69:fopen('pki/prime256v1.pem','w')
0:error:2006D080:BIO routines:BIO_new_file:no such file:crypto/bio/bss_file.c:76:

Traceback (most recent call last):
  File "genconfig.py", line 197, in <module>
    main()
  File "genconfig.py", line 193, in main
    gen_cert(sig)
  File "genconfig.py", line 63, in gen_cert
    common.run_subprocess([OPENSSL, "ecparam", "-name", "prime256v1", "-out", os.path.join(PKIPATH, "prime256v1.pem")])
  File "/home/ubuntu/oqs-demos/nginx/fulltest/common.py", line 47, in run_subprocess
    assert False, "Got unexpected return code {}".format(result.returncode)
AssertionError: Got unexpected return code 1
```

--- a/nginx/fulltest/README.md
+++ b/nginx/fulltest/README.md
@@ -39,7 +39,7 @@ Leave away the `install` option to first do a test-build, e.g., if executed on a
 
 #### First-time execution
 
-`python3 genconfig.py` generates all required QSC certificates. Execute this at the first installation and/or if/when algorithms supported by liboqs have changed since the last installation. **This script overwrites an existing installation's configuration files. Use with care on a live server.**
+`python3 genconfig.py` generates all required QSC certificates. Execute this at the first installation and/or if/when algorithms supported by liboqs have changed since the last installation. **This script overwrites an existing installation's configuration files. Use with care on a live server.** Be sure to have the directory `pki` available/created: For example environmental setup, see e.g., [package.sh](package.sh).
 
 #### Activation
 

--- a/nginx/fulltest/genconfig.py
+++ b/nginx/fulltest/genconfig.py
@@ -56,6 +56,10 @@ def gen_cert(sig_alg):
                                      '-config', OPENSSL_CNF])
            print("New root cert residing in %s." % (os.path.join(CAROOTDIR, "CA.crt")))
 
+   # first check whether we already have a PKI dir; if not create it
+   if not os.path.exists(PKIPATH):
+           os.mkdir(PKIPATH)
+
    # now generate suitable server keys signed by that root; adapt algorithm names to std ossl 
    if sig_alg == 'rsa3072':
        ossl_sig_alg_arg = 'rsa:3072'

